app/javascript/components/widgets/numbers/panel.jsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import Wrapper from '../../panel-wrapper';
+import FallingBlocks from './fallingblocks';
 
-export const Numbers = () => <Wrapper />;
+export const Numbers = () => (
+  <Wrapper>
+    <FallingBlocks pr={143} commit={160} slack={500} />
```

Comment:
This can wrapped with a graphql query. Have a look at `app/javascript/components/widgets/numbers/carousel.jsx`

There are rake tasks to simulate pr and commits:
```
rake simulate:commit[count]             # Simulates a new commit on github
rake simulate:pull_request              # Simulates opening a new pull request on github

```